tools/wasi/wasi.hpp

```diff
@@ -26,6 +31,18 @@ std::optional<bytes> load_file(std::string_view file, std::ostream& err) noexcep
 /// @todo Make noexcept.
 bool load_and_run(int argc, const char** argv, std::ostream& err);
 
+/// Instantiates a module that imports WASI functions.
+std::unique_ptr<Instance> instantiate(bytes_view wasm_binary);
```

Comment:
What is the reason behind splitting?

Reply:
To access instance in tests after `run`.